spec.bs

```diff
@@ -2319,7 +2632,7 @@ of the following global objects:
     1. Let |reportURL| be |global|'s [=InterestGroupReportingScriptRunnerGlobalScope/report url=]
     1. If |reportURL| is failure, set |reportURL| to null.
     1. Let |macroMap| be |global|'s [=InterestGroupReportingScriptRunnerGlobalScope/reporting macro map=]
-      if |functionName| is "`reportWin`", null otherwise.
+      if |functionName| is "`reportWin`" or "`reportContextualWin`", null otherwise.
     1. Return « |resultJSON|, |reportURL|,

```

Comment:
This may yet change to reportAdditionalBidWin. I'll update this comment when we've decided one way or the other for certain. Thanks.